app/policies/comment_policy.rb

```diff
@@ -0,0 +1,19 @@
+class CommentPolicy < ApplicationPolicy
+  def create?
+    user.present?
+  end
+
+  def edit?
+    user.present?
+  end
+
+  def update?
+    user.present?
```

Comment:
Maybe user has to be the author of the task?

Reply:
Sure